### DISCUSSION TITLE
How can I customize the connector in Wagmi v2?

### DISCUSSION THREAD

**Discussion by Jaewoneeee** [wevm/wagmi#3535] [Q&A] [answered]
I'm currently using a custom connector in Wag 1.4.12, following the instructions in the [link](https://0.3.x.wagmi.sh/examples/custom-connector).

 What should I do in V2, is there any official documentation on this?

**Comment by tmm** [comment, marked as answer]
Here's the info on creating connectors for v2: https://wagmi.sh/dev/creating-connectors You should be able to port the v1 connector over.
